packages/platforms/accton/x86-64/as7535-28xb/modules/builds/x86-64-accton-as7535-28xb-sys.c

```diff
@@ -329,12 +333,20 @@ static int sysfs_eeprom_cleanup(struct kobject *kobj,
 	return 0;
 }
 
-static struct as7535_28xb_sys_data *as7535_28xb_sys_update_fpga_ver(void)
+static struct as7535_28xb_sys_data *as7535_28xb_sys_update_ver(
+				struct sensor_device_attribute *attr)
 {
 	int status = 0;
 
 	data->valid = 0;
-	data->ipmi_tx_data[0] = 0x60;
+
+	if(attr->index == FPGA_VER)
```

Comment:
For consistency, the rest of the file also seems to add a space after `if` and `else if`.